topo_drag/README.md

```diff
@@ -0,0 +1,35 @@
+# `topo_drag.res.nc`
+
+This file was generated by running
+
+```console
+fregrid --standard_dimension --input_mosaic ../grid/C96_mosaic.nc --input_file topo_drag.res --interp_method conserve_order2 --remap_file ../grid/.fregrid_remap_file_288_by_180.nc --nlon 288 --nlat 180 --scalar_field t11,t21,t12,t22,hmin,hmax --output_file topo_drag.res.nc
+```
+
+The input data was obtained from the AM4 model and can be found [here](https://data1.gfdl.noaa.gov/nomads/forms/am4.0/)
+
+The `fregrid` tool can be found [here](https://github.com/NOAA-GFDL/FRE-NCtools/tree/main)
```

Comment:
I think this is a little misleading. Yes, `fregrid` is needed, but it is just a regridding tool. The important thing about this file is the input file, `topo_drag.res`. 

@szy21 could you suggest a short description for this artifact?

Reply:
Something like this would be good:

`topo_drag.res.nc` contains all relevant information about the topography, including amplitude, variance, orientation, and anisotropy. It includes the matrix tensor T, as well as hmax and hmin defined in this [paper](https://journals.ametsoc.org/view/journals/atsc/62/7/jas3496.1.xml).